clean_project/construct_dict.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+"""
+Construct a JSON file used to indexing all words in a config_file.yaml
+"""
+import argparse
+import codecs
+import json
+import os
+import yaml
+
+from collections import defaultdict
+from clean_file import sanitize
+
+
+def construct_dictionary(yaml_path, path_json_file):
+    """
+    Construct the dictionary, given a configuration files
+    :param yaml_path: Yaml's path that contains a list of files that will be cleaned
+    :param path_json_file: Path of JSON file (indexing)
+    :return: JSON File
+    """
+    output = []
+    with open(yaml_path) as config_file:
+        data = yaml.load(config_file)
+        if data is not None:
+            for file_dirty in data['files_path']:
+                with codecs.open(file_dirty, 'r', 'utf-8') as source_file:
+                    memory = defaultdict(list)
+                    memory["file_name"].append(file_dirty)
+                    for i, line in enumerate(source_file, 1):
```

Comment:
because the first line is line 1. It right start to define the first line as line 1